setup.py

```diff
@@ -67,8 +78,9 @@ def run(self):
     ext_modules = [ Extension( "PyNormaliz_cpp",
                               [ "NormalizModule.cpp" ],
                               extra_compile_args=['-std=c++11'],
+                              libraries=libraries,
                               **extra_kwds) ],
     
     package_data = {'': [ "COPYING", "GPLv2", "Readme.md" ] },
-    cmdclass = {'test': TestCommand},
+    cmdclass = {'build': build, 'test': TestCommand},
```

Comment:
Do you want to do `build` here` C dependencies are usually build via `build_ext`, so maybe we should overload that command instead? I am not sure though,

Reply:
It is also a possibility but apparently `pip` does not call `build_ext`. I am trying to understand this...

Reply:
Sorry, pip does call `build_ext`.

Reply:
What is confusing is that `pip` creates a temporary directory, copy all files, and then run the various setup steps.

Reply:
Changed in 6a60f7d